docs/edit_first_game_server.md

```diff
@@ -1,11 +1,12 @@
-# Getting Started
+# Quickstart Edit a Game Server
```

Comment:
In the future - please keep edits like this in separate PRs. Something like this would be quick to push through, but this PR is being held up by other things.

Reply:
Definitely.  I will do so in the future.